packages/matter.js/src/device/AutoDevice.ts

```diff
@@ -177,10 +185,24 @@ class WiringContext {
 
     // Implement device commands
     private wireCommand(command: CommandModel) {
-        this.deviceProto[command.invoker] = ignore;
+        // Install the default handler
+        const defaultHandler = this.cluster.defaultServerHandlers[command.handler];
+        if (defaultHandler) {
+            // TODO - this may not work or may otherwise change pending PR
+            // conversations
+            this.deviceProto[command.invoker] = function(request: any, { session, message, endpoint }: CallContext) {
```

Comment:
and we need to get the typing right ... Re "any"

Reply:
This is implementing the interface method...  The type is correct in the public signature.